eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/ConnectionHandler.java

```diff
@@ -85,13 +85,6 @@ public void onConnectionInit(Event event) {
         connectionProperties.put(AmqpConstants.PLATFORM, ClientConstants.PLATFORM_INFO);
         connectionProperties.put(AmqpConstants.FRAMEWORK, ClientConstants.FRAMEWORK_INFO);
 
-        final String userAgent = EventHubClientImpl.USER_AGENT;
```

Comment:
Why was this code removed? We trace this value on the service side so we know what client implementation and version is connecting to us!

Reply:
Clarification: we do trace this string, but it is for libraries that build on top of the client. Note that EventHubClientImpl.USER_AGENT is public and can be set by anybody, meaning there is no guarantee that it will still be null by the time this code is executed.